### PR TITLE
removed module.js from package.json

### PR DESCRIPTION
**Pull request by CluEleSsUK**
imported Buffer everywhere so browsers don't need a polyfill

### CODE REVIEW

**Comment by CluEleSsUK** [comment]
The packaging is a little magical: https://github.com/drand/tlock-js/blob/0638fc39a0b8ad73990cdd430f9661b7ee6768ef/.github/workflows/build.yml#L27

We do it via a github action that moves some stuff around. Previously we used parcel which was a bit of a pain mapping dist folder to be packaged and such. I'd love to move to esbuild and just build a single js file, but unfort that'd be a breaking change for anyone using imports such as "tlock-js/age"

**Comment by wighawag** [comment]
Thanks, I ll try that

Been playing with typescript/js for a while but still not sure best way to organize build pipeline but for now I am quite happy with [tsup](https://tsup.egoist.dev/) (which use esbuild behind the scene) and it supports multiple entries/output, so maybe it can work for you

**Comment by wighawag** [comment]
Any update on this ?
I am now integrating tlock-js in a production build and would like to use the latest tlock-js but still need to patch it.

**Comment by CluEleSsUK** [comment]
sorry, @AnomalRoil is out until tomorrow - he should be around to approve and merge then!

File: src/index.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ import {createTimelockEncrypter} from "./drand/timelock-encrypter"
 import {decryptAge, encryptAge} from "./age/age-encrypt-decrypt"
 import {decodeArmor, encodeArmor, isProbablyArmored} from "./age/armor"
 import {createTimelockDecrypter} from "./drand/timelock-decrypter"
-import {defaultChainInfo, defaultChainUrl, MAINNET_CHAIN_URL, TESTNET_CHAIN_INFO, TESTNET_CHAIN_URL} from "./drand/defaults"
+import {defaultChainInfo, defaultChainUrl, MAINNET_CHAIN_URL, TESTNET_CHAIN_URL} from "./drand/defaults"
```
Comment (AnomalRoil) [inline src/index.ts:15]: Why the removal?

**Review by AnomalRoil — APPROVED** [review]

File: src/index.ts (hunk 1 above)
Comment (CluEleSsUK) [reply src/index.ts:15]: updated and fiddled some testnetclient stuff

**Pull request merged**